libsplinter/src/oauth/profile/openid.rs

```diff
@@ -55,10 +56,38 @@ impl ProfileProvider for OpenIdProfileProvider {
             }
         }
 
-        let user_profile = response
+        let mut user_profile = response
             .json::<OpenIdProfileResponse>()
             .map_err(|_| InternalError::with_message("Received unexpected response body".into()))?;
 
+        // If azure openid is being used for authentication make a call to the
+        // microsoft graph api endpoint with the access token to retrieve the
+        // binary data for the authenticated user's profile photo
+        if self.userinfo_endpoint.contains("graph.microsoft.com") {
+            let picture_response = match Client::builder()
+                .build()
+                .map_err(|err| InternalError::from_source(err.into()))?
+                .get("https://graph.microsoft.com/beta/me/photo/$value")
+                .header("Authorization", format!("Bearer {}", access_token))
+                .send()
+            {
+                Ok(image) => Some(encode(
+                    image
+                        .bytes()
```

Comment:
Probably would do the same with this error, too

Reply:
Fixed